packages/sw-build-webpack-plugin/index.js

```diff
@@ -0,0 +1,65 @@
+const swBuild = require('sw-build');
+const path = require('path');
+const SERVICE_WORKER_NAME = 'sw.js';
```

Comment:
I agree with Matt that defaults might be better defined in sw-build otherwise we might set folks up for the wrong expectations. I'd imagine most folks are going to copy/paste whatever code snippets for using the Webpack plugin we're going to include in our docs and that might be a better place to include both of these bits (I think that would be useful).